### DISCUSSION TITLE
Error when building on html5

### DISCUSSION THREAD

**Discussion by NikRetaNCAM** [HaxeFlixel/flixel#2443] [Q&A]
It just responds me with:
![image](https://user-images.githubusercontent.com/60806448/141049336-611335eb-51f1-41ce-b1c4-76aece2b7dec.png)

How to fix that??
(note: it does build on windows)

**Comment by RichardBray** [comment]
That's interesting, don't think I've seen that error before. It would be good to know how you installed lime/openfl/flixel. I'm not a Windows expert but I know things are usually installed on the C drive and you're testing the game on D. Does that have anything to do with it?

**Reply by NikRetaNCAM** [reply]
> That's interesting, don't think I've seen that error before. It would be good to know how you installed lime/openfl/flixel. I'm not a Windows expert but I know things are usually installed on the C drive and you're testing the game on D. Does that have anything to do with it?

my username has russian letters so i installed everything on D

**Reply by RichardBray** [reply]
Hmmm, then I'm not sure. If you're comfortable sharing the code to your project I could try it on my machine.

**Reply by NikRetaNCAM** [reply]
> 

i think i can share the code but how tho

**Reply by RichardBray** [reply]
If you create a public github repo, I'll be able to clone the code and run it løcally.

**Reply by NikRetaNCAM** [reply]
> If you create a public github repo, I'll be able to clone the code and run it løcally.

in a few minutes

**Reply by NikRetaNCAM** [reply]
> If you create a public github repo, I'll be able to clone the code and run it løcally.

https://github.com/NicklGames/citu-source

took me some time to understand how gitHub works

**Reply by RichardBray** [reply]
Just tested it, seems to work fine for me:

<img width="1157" alt="Screenshot 2021-12-13 at 18 49 43" src="https://user-images.githubusercontent.com/1377253/145870874-fe5ae114-f86a-4428-a9ad-58a2ecf979df.png">

Do you still have issues with HTML5?

**Reply by Geokureli** [reply]
i think it's an issue with their directory, I never see projects on D, and their C drive has russian characters

**Reply by NikRetaNCAM** [reply]
> Just tested it, seems to work fine for me:
> 
> <img alt="Screenshot 2021-12-13 at 18 49 43" width="1157" src="https://user-images.githubusercontent.com/1377253/145870874-fe5ae114-f86a-4428-a9ad-58a2ecf979df.png">
> 
> Do you still have issues with HTML5?

hol' up. i'll test it

nothing changed.

**Comment by giuppe** [comment]
> It just responds me with: ![image](https://user-images.githubusercontent.com/60806448/141049336-611335eb-51f1-41ce-b1c4-76aece2b7dec.png)
> 
> How to fix that?? (note: it does build on windows)

What does
`lime test html5 --verbose`
output?

**Reply by NikRetaNCAM** [reply]
> > It just responds me with: ![image](https://user-images.githubusercontent.com/60806448/141049336-611335eb-51f1-41ce-b1c4-76aece2b7dec.png)
> > How to fix that?? (note: it does build on windows)
> 
> What does `lime test html5 --verbose` output?

same output.

**Reply by RichardBray** [reply]
So I installed lime with [lix](https://github.com/lix-pm/lix.client) which runs off nodejs. Lix is a bit more flexible, if you're comfortable with that maybe we could try that approach?
